### DISCUSSION TITLE
Submitting Content around E5 Capabilities

### DISCUSSION THREAD

**Discussion by FrankMcGovern** [MicrosoftDocs/microsoft-365-community#324] [General]
I am here looking to add my content after being suggested this repo by @sympmarc.

The article is at: https://frankmcg.com/2020/09/purchased-microsoft-365-e5-now-what/

I'm assuming we don't want to open an Issue to get anything added, but rather a PR and am new to PR's. Unsure of the best path forward.

**Comment by sympmarc** [comment]
Hi, Frank. Thanks for coming on over!

If you'd like to learn how to add your article with a PR, you could read through the [Adding Content](https://github.com/MicrosoftDocs/microsoft-365-community/wiki/Adding-Content) article and proceed.

If you'd prefer help, I can have @hugoabernier and/or @PopWarner get in touch. They run a monthly call to help people get up to speed with the process.

Or, if you'd rather just not deal with it, I could copy the content from your blog and post it myself. But then you wouldn't get your GitHub handle associated with the contribution - which you should get credit for!

**Reply by FrankMcGovern** [reply]
That helps. That was the link I missed in the README. It may be wise to link under the Contributing section too.

I will get working on building out my blog in markdown so we can merge!

The biggest question is what area will this go in? Do you want me to fork areas and make changes to then submit PR's or should I create my own rebo with content and then we can merge it?

Should I do it all in one document or should this be broken out by each individual product?

I could probably use some guidance on formatting, proper way to break it out, etc.

**Reply by sympmarc** [reply]
I was thinking either in the Security section or Microsoft 365 Better Practices. I'll make the edits to the TOC/index when we're at that point; I should add that to the docs!

If you think you'll be contributing more than one article, take a fork and do PRs. If GitHub isn't your thing, you can just add a new file in place.

You article is indeed massive. :) If you think you have some way to split it into logical pieces, then go for it. We can also iterate on it together one there's a copy in the repo.

**Comment by sympmarc** [comment]
@FrankMcGovern Still interested in contributing?

**Reply by FrankMcGovern** [reply]
@sympmarc - I am! Sorry for the delay. I made a company change and have been quite busy! When is the next monthly cadence call? It would be worth me getting into that for some initial assistance.

**Reply by sympmarc** [reply]
No worries at all. Contributing here is no one's full time job. We don't have monthly calls specifically about this repo (maybe we should?). Are there specific questions I can help with?

**Reply by FrankMcGovern** [reply]
@sympmarc - I quick reviewed the link you shared around getting started. I think I may be able to figure it out. I will find time by year's end!

**Reply by sympmarc** [reply]
Great! Don't hesitate to reach out with questions.

We talked about your question about monthly calls above on the PnP Team meeting today. We're thinking about some sort of "office hours" every month for this effort as well as some of the others, probably as part of the [SharingIsCaring](https://pnp.github.io/sharing-is-caring/) idea. Thanks for raising the idea! @hugoabernier @popwarner
